src/main/resources/processes/FOI_ACCEPTANCE.bpmn

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think Camunda and casework should be updated in a single call the BpmnService. That seems to be the existing pattern.

Reply:
hmm, I am having second thoughts. Not sure the best pattern here.

Reply:
Would you suggest creating a new method in the BPMNService to handle this?

Reply:
That's what I was wondering. Also shouldn't we also be wiping the 'Directorate' Camunda process variable as we clear this in the casework db?

Reply:
I've created a new method to do this in one task and clear both variables.